### DISCUSSION TITLE
Attaching Screenshots to the HTML Reports in case of Failures

### DISCUSSION THREAD

**Discussion by ayanghoshniq** [hrcorval/behavex#111] [Q&A] [answered]
Adding HTML report screenshots to documentation is a  part of 1.5.6 .
But I do not see any documentation how to attach screenshots in case of failures while using behavex.

Is it supported or not ?

Running as per -- behavex .\features\CustomerService.feature -o allure_reports

Below is the folder structure
![image](https://github.com/hrcorval/behavex/assets/145746393/b9954a0f-d9ae-4587-8a25-252e61865347)

@hrcorval Can you please help here ?

**Comment by hrcorval** [comment, marked as answer]
Hi @ayanghoshniq, you have a specific feature for adding screnshots to the HTML report, in which you can see all the attached screenshots or images in a gallery. Please take a look at the following documentation to get more information about how to attach and see the screenshots: https://github.com/hrcorval/behavex?tab=readme-ov-file#attaching-images-to-the-html-report

I hope it helps. Any feedback on this is really welcome :)

**Comment by vibincwilson** [comment]
[behavex-images] The provided binary is not a valid image, or could not be converted to PNG: module 'numpy' has no attribute 'float'.
`np.float` was a deprecated alias for the builtin `float`. To avoid this error in existing code, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here

Getting this error while attaching PNG

**Reply by hrcorval** [reply]
Hi @vibincwilson, the issue you were experiencing is already fixed in latest version, as you suggested, we started using np.float64, as we see it is not impacting the library functionality (thanks for the suggested fix)
Please, keep us posted on that.
Thanks!
